jupyterhub/app.py

```diff
@@ -272,6 +272,10 @@ class JupyterHub(Application):
     def _template_paths_default(self):
         return [os.path.join(self.data_files_path, 'templates')]
 
+    base_templates = Bool(False,
```

Comment:
I think it's okay for the base templates to always be available. This doesn't need to be configurable.

Reply:
This makes sense. Much like extending templates with Sphinx.